### PR TITLE
Ia styling home page

### LINKED ISSUE #47: Style the home page to look  like the figma design
## Description

Style the Home page to look like the  figma design

## Acceptance Criteria
- [ ] The home page is styled  to have a great look as the [figma design](https://www.figma.com/file/WDYjT265wEzdRlEXKTrclx/smart-Shopping-List?node-id=9%3A112&t=NbWcc8vO8RbuKgAy-0)

### PR DESCRIPTION
**Pull request by mide358**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

The home page have been styled to look like the figma design

## Related Issue
Closes #47 

## Acceptance Criteria
 [x] The home page is styled  to have a great look as the [figma design](https://www.figma.com/file/WDYjT265wEzdRlEXKTrclx/smart-Shopping-List?node-id=9%3A112&t=NbWcc8vO8RbuKgAy-0)

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Desktop
image

### Mobile
image

### Desktop
image

### Mobile
image

## Testing Steps / QA Criteria

1. From your terminal, do `git checkout main`
2. Pull down this branch with `git pull origin ia-styling-home-page`
3. Checkout the branch with `git checkout ia-styling-home-page`
4. See changes in desktop and mobile version

### CODE REVIEW

**Review by vicowiti — CHANGES_REQUESTED** [review]
Great work, I think we should maintain a max height of 100vh. Using styles like `h-screen` or h-[100vh]. so that the user sees all parts of the landing page .  Since its a global issue, we should work on it when refactoring

**Review by vicowiti — APPROVED** [review]

File: src/views/Home.jsx [hunk 1]
```diff
@@ -10,25 +10,60 @@ export function Home({ onClick, onChange, handleForm }) {
...
-				Create a new list
+		<div
+			className="home mt-20 flex flex-col
+		items-center justify-center "
+		>
+			<button
+				onClick={onClick}
+				className="create-list-btn bg-[#C2410C] text-white w-[243px] lg:w-[534px] h-[46px] lg:h-[90px]
+				rounded-[10px] font-poppins font-bold text-[20px] lg:text-[32px] leading-[30px] mb-16 border border-1 border-black
+				"
```
Comment (klezi10) [inline src/views/Home.jsx]: I think 32px for text is too big. This is what I see in the browser
image

**Review by klezi10 — CHANGES_REQUESTED** [review]

**Review by klezi10 — APPROVED** [review]

**Review by michhub-dev — APPROVED** [review]

File: src/views/Layout.css [hunk 2]
```diff
@@ -26,6 +26,7 @@
 
 .Layout-header > h1 {
```
Comment (redapy) [inline src/views/Layout.css:27]: Just a convention, the class names start with lowercase.

**Review by redapy — CHANGES_REQUESTED** [review]
Great Work! I have a non-blocking note/suggestion, try to not use extra classes for styles when you have a unique HTML element that you can use as a CSS selector.
Here's some examples: 
- `.Layout-header` => header
- `.Layout-header > h1` => header > h1
-  `.Layout-main` => main
- `.Nav` => nav
- `.Nav-link` => nav > a.   (here it's okay to use a class, I'm just giving examples)

**Review by redapy — APPROVED** [review]

**Pull request merged**